packages/stark-build/config/build-utils.js

```diff
@@ -31,13 +31,20 @@ function readTsConfig(tsConfigPath) {
 
 function getEnvironmentFile(environment) {
 	if (typeof environment === "string") {
-		let fileName = helpers.root("src/" + getAngularCliAppConfig().environments[environment]);
+		let fileName = helpers.root("src/environments/environment.ts");
+		let fileNameAlt;
+		let angularCliEnvConfig = getAngularCliAppConfig()["architect"]["build"]["configurations"][environment];
```

Comment:
Again, I think we should just use the dot syntax whenever possible...

Reply:
done